dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/TeTaValidationTest.java

```diff
@@ -230,11 +230,24 @@ void testTeaInvalidImage()
         TrackerImportParams trackerImportParams = createBundleFromJson(
             "tracker/validations/te-program_with_tea_invalid_image_value.json" );
         TrackerImportReport trackerImportReport = trackerImportService.importTracker( trackerImportParams );
-        assertEquals( 2, trackerImportReport.getValidationReport().getErrors().size() );
+        assertEquals( 1, trackerImportReport.getValidationReport().getErrors().size() );
         assertThat( trackerImportReport.getValidationReport().getErrors(),
-            hasItem( hasProperty( "errorCode", equalTo( TrackerErrorCode.E1085 ) ) ) );
```

Comment:
Does the import payload still have 2 errors? If so (only if so 😋 ) you changed behavior away from what I understood the goal of NTI validation was. To provide all errors we can so the user can fix all errors at once, then try the import again and expect it to work. @stian-sandvold is that the goal of NTI validation 😄 ?